README.md

```diff
@@ -266,6 +267,15 @@ const credentials = {
 ```
 This scraper supports fetching transaction from up to one year.
 
+## Beinleumi & Massad
+These scrapers are essentially identical and expect the following credentials object:
+```node
+const credentials = {
+  username: <user name>,
+  password: <user password>
+};
+```
+
```

Comment:
let's move this up, next to the rest of the banks - I would put it after Mizrahi scraper

Reply:
Done.